meltano.yml

```diff
@@ -13,14 +13,20 @@ plugins:
     - discover
     settings:
     - name: access_token
+      value: ${TAP_LINKEDIN_ADS_ACCESS_TOKEN}
```

Comment:
@NeilGorman104 these default value aren't necessary. If someone sets the environment variable `TAP_LINKEDIN_ADS_ACCESS_TOKEN` using the `<PLUGIN_NAME>_<SETTING_NAME>` syntax then it will already override whatever is configured for this setting without needing `value: ${TAP_LINKEDIN_ADS_ACCESS_TOKEN}`. See https://docs.meltano.com/guide/configuration#configuring-settings

Does that make sense?

Reply:
Hey Pat. This makes sense. This change was made to fix a 401 error in the CI test. However, we have not faced this issue previously when the env values weren't set in meltano.yml so there is likely another way.

Will revert this change.